core/controller/src/main/scala/whisk/core/controller/Backend.scala

```diff
@@ -67,10 +66,9 @@ object WhiskServices {
      * and returns the HTTP response from the load balancer as a future
      */
     def makeLoadBalancerComponent(config: WhiskConfig, timeout: Timeout = 10 seconds)(
-        implicit as: ActorSystem): (LoadBalancerReq => Future[Unit], () => JsObject, (ActivationId, FiniteDuration, TransactionId) => Future[WhiskActivation]) = {
+        implicit as: ActorSystem): LoadBalancerService = {
         val loadBalancer = new LoadBalancerService(config, InfoLevel)
-        val requestTaker = (lbr: LoadBalancerReq) => { loadBalancer.publish(lbr._1)(lbr._2) }
-        (requestTaker, loadBalancer.getInvokerHealth, loadBalancer.queryActivationResponse)
+        loadBalancer
```

Comment:
I think this can be just

```
def makeLoadBalancerComponent(config: WhiskConfig)(implicit as: ActorSystem) =
        new LoadBalancerService(config, InfoLevel)
```